packages/web-components/public/theme-switch.ts

```diff
@@ -0,0 +1,13 @@
+import { teamsDarkTheme, teamsLightTheme, webDarkTheme, webLightTheme } from '@fluentui/react-theme';
+import { setTheme } from '../src/theme';
+
+const themes = {
+  'web-light': webLightTheme,
```

Comment:
For now, there are no theme exports from web components.
If we decide to go this way, we need to discuss the export contract.

Reply:
Is this for Storybook only? Named exports are treeshakable but an object is not, so all themes will always be included here.

Reply:
Correct, storybook only for now.
Once we know the packages structure, we will decide whether partners will import the theme directly from `theme` package or `web-components` will re-export the themes and how.

Reply:
We can discuss later - may be ideal to export from the package though given we’re using DesignToken.